### ISSUE TITLE
Allow setting a specific jail uuid when creating a jail

### ISSUE THREAD

**Issue by Licenser** [iocage/iocage#65]
# Short

It would be nice to have a option like:

```
iocage create uuid=<put my uuid here> ...
```

This would then allow instead of doing this:

https://github.com/iocage/iocage/blob/master/iocage/lib/ioc_create.py#L48

using the supplied UUID.

# Long

I'm trying to port parts of [Project-FiFo](https://project-fifo.net) over to FreeBSD and currently am using iocage to interact with the jails. FiFo generates a UUID before starting the zone (or jail) creation process to allow tracing and displaying things in an asyncronous manner. This is problematic when the `create` command can't enforce a specific UUID for the zone. The current approach I found is using the tag 'fifo:<fifo-uuid>' which in itself is slightly problematic as it makes the output of `iocage list` less human readable.

**Comment by skarekrow** [comment]
Closed by #68
